### PR TITLE
Ticket 1397: equal heights display in multimedia row

### PR DESCRIPTION
**Pull request by arcepaul**
#1397 : equalheights was working correctly but feature card wasn't filling the extra column height. 
- added height 100% to feature card under equalheight column

### CODE REVIEW

**Comment by arcepaul** [comment]
Adding @BrendanBeltzNIH  b/c he was enlightening me on the history behind this...

**Comment by BrendanBeltzNIH** [comment]
I would suggest replacing the slot item wrapper div rather than changing the the stylesheet if possible.

**Comment by arcepaul** [comment]
ODE example available at:
http://ncigovcdode86.prod.acquia-sites.com/about-cancer

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--field-mm-feature-card.html.twig [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <div id="nvcgSlLayoutFeatureB" class="equalheight small-4 columns card">
-  {% for item in items %}
-    {{- item.content -}}
-  {% endfor %}
+  <div class="slot-item only-SI">
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--field-mm-feature-card.html.twig]: Can you move this into the for each? The idea is that `nvcgSlLayoutFeatureB` is the slot, and each content item in the slot gets the wrapper.

`slot-item` is given to all items in the container.
`first-SI` is given to the first element when there is more than one
`last-SI` is given to the first element when there is more than one
`only-SI` is given when it is the only element.

Maybe if someone looks at the CSS to find first/only/last and sees nothing then we can dispatch with that class.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss [hunk 2]
```diff
@@ -523,10 +523,10 @@ parent accordion in all breakpoints. */
 }
 .multimedia-feature-card,
 .multimedia .feature-card {
-  @extend %autoheight-fix;
-  
+  //@extend %autoheight-fix;
   a {
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss]: I was under the assumption based on @BrendanBeltzNIH comment that adding the class back in would not need a change to Homelanding.scss. Was that incorrect?

Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss]: True, it does no harm but just ends up being bloat .  This is just rolling it back to what it was already.  I believe Brendan just wanted to add the wrapper div back to the template and the style is already handled with what's there now.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--field-mm-feature-card.html.twig (hunk 1 above)
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field--paragraph--field-mm-feature-card.html.twig]: Ok, I have moved the slot-item wrapper into the for-each for the feature card and the media item.  The only-SI doesn't do anything in the context of the multimedia so I remvoed it.  I found no reference to the other SIs in the multimedia styles

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss (hunk 2 above)
Comment (bryanpizzillo) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss]: Cool!

**Pull request merged**